fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -247,6 +248,25 @@ public String getObjectsAsString(final Graph graph,
     }
 
     /**
+     *
+     * @param graph
+     * @param subject
+     * @return
+     */
+    public int getNumChildren(final Graph graph, final Node subject) {
+        LOGGER.trace("Getting number of children: s:{}, g:{}", subject, graph);
+        final Iterator<Node> iterator = listObjects(graph, subject, HAS_CHILD_COUNT.asNode());
```

Comment:
The pattern between the javascript and ViewHelpers is to pass in a `graph` vs. a `Resource`, which is required by `GraphUtils::getStringValue`. If you have a pull-request against this branch that uses `GraphUtils.getStringValue`, I would be happy to merge it into this work. Otherwise, I would suggest moving forward with this impl as is.